src/main/java/games/strategy/engine/config/client/remote/UrlRedirectResolver.java

```diff
@@ -0,0 +1,43 @@
+package games.strategy.engine.config.client.remote;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import com.google.common.annotations.VisibleForTesting;
+
+/**
+ * Module that follows URLs and any redirects and determines the final URL to use.
+ */
+class UrlRedirectResolver {
```

Comment:
It's recommended putting `@deprecated` in the javadoc and `@Deprecated` as annotation